packages/dashboard/src/layout/LayoutUtils.ts

```diff
@@ -597,12 +602,8 @@ class LayoutUtils {
     const oldContentItem = LayoutUtils.getContentItemInStack(stack, config);
     const maintainFocusElement = document.activeElement; // attempt to retain focus after dom manipulation, which can break focus
     if (oldContentItem) {
-      if (oldContentItem.isComponent) {
-        // container property exists on a contentItem if contentItem is a component,
-        // however, this is not included in the types for a contentitem, thus the casting
-        ((oldContentItem as unknown) as {
-          container: { close: (isOverwrite?: boolean) => void };
-        }).container.close(isOverwrite);
+      if (isComponent(oldContentItem)) {
+        oldContentItem.container.close(closeOptions ?? undefined);
```

Comment:
The `?? undefined` is unnecessary - `closeOptions` is already `CloseOptions | undefined`, so this null coalescing just falls back from `undefined` to `undefined`.
Should just be `oldContentItem.container.close(closeOptions);`